packages/cape/src/methods.ts

```diff
@@ -360,6 +365,14 @@ export abstract class Methods {
   private async verifyAttestationDocument(message: string, checkDate?: Date): Promise<AttestationDocument> {
     const doc = parseAttestationDocument(message);
 
+    if (this.nonce != undefined) {
+      const b64Nonce = Buffer.from(this.nonce).toString('base64');
+      const b64DocNonce = Buffer.from(doc.nonce).toString('base64');
+      if (b64DocNonce != b64Nonce) {
```

Comment:
```suggestion
      if (b64DocNonce !== b64Nonce) {
```
Always use strict comparisons when possible 🙂